runtime/autoload/download.lua

```diff
@@ -0,0 +1,41 @@
+basepath = "http://ftp.vim.org/pub/vim/runtime/spell/"
+spell_home = "~/.config/nvim/spell/"
+
+--[
+-- Checks if a file or directory exists.
+-- ]
+function exists(path)
```

Comment:
Nvim already has very robust solution for this (`:help filereadable()`, `:help isdirectory()`), there is no reason to re-implement it in Lua.